app/theme/client/main.css

```diff
@@ -58,3 +58,429 @@
 
 /* RTL */
 @import 'imports/general/rtl.css';
+
```

Comment:
There is a lot of indentation problems here, your code has spaces and tabs.
Also, add this changes to the `livechat.css` file, not the `main.css`.

Reply:
I have created livechat css and solved the issue of messing issue but for the indention issue on css i was trying to resolve but i wont be able to .  I am now only using indention of tab only but it always showing needed 1 indention weather i have added indention or not it always shows add indention i dont know how to resolve this issue

Reply:
Remove the blank line

Reply:
removed the blank lines from the code